resources/lang/de/admin/nests.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Its not documented, but all of the license headers can be removed from these files. We're moving towards that on all files, I just haven't done a gloabl search and replace yet.

So lines 2 - 9 can be removed.